### ISSUE TITLE
Misc changes in notification icon color

### ISSUE THREAD

**Issue by asudoh** [carbon-design-system/carbon#3443]
From @jeanservaas at #3377:

- [ ] need black fill for error icon for accessibility

- [ ] need black fill for check mark icon for accessibility

<img width="650" alt="image" src="https://user-images.githubusercontent.com/43144260/61400503-02614280-a896-11e9-9fd1-545935e177bc.png">

**Comment by asudoh** [comment]
@jeanservaas Unlike the warning icon, seems that the error/checkmark icons don't have `<path>` for the inner content to fill with black color. Would your team be able to change the icon for that? Thanks!

**Comment by jeanservaas** [comment]
@chrisconnors-ibm I remember you dealing with the icon adornment issue in the past (i think for exclaimation mark and for the Do / Don't examples) -- what steps do we have to take to make this happen for the other two?

**Comment by jeanservaas** [comment]
Not stale — we need to do this for consistency and accessibility.

**Comment by chrisconnors-ibm** [comment]
notStale-bot

**Comment by tw15egan** [comment]
Closed in favor of https://github.com/carbon-design-system/carbon/issues/3452
